src/main/java/pl/wrapper/parking/facade/domain/DummyController.java

```diff
@@ -0,0 +1,34 @@
+package pl.wrapper.parking.facade.domain;
+
+import jakarta.servlet.http.HttpServletRequest;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+import pl.wrapper.parking.infrastructure.error.*;
+import pl.wrapper.parking.infrastructure.error.Error;
+
+@RestController
+@RequiredArgsConstructor
+public class DummyController extends HandleResult {
+    private final DummyService dummyService;
+
+    @GetMapping("/id/{id}")
+    public ResponseEntity<String> getParkingOccupancyByParkingIdValid(@PathVariable("id") Long id) {
+        boolean willSucceed = true;
+        Result<Long> result = dummyService.dummyGetParkingBySymbol(id, willSucceed);
+        return handleResult(result, HttpStatus.OK, "/id/" + id);
+    }
+
+    @Override
+    protected ErrorWrapper getInfoByError(Error error, String uri, HttpStatus onSuccess) {
```

Comment:
let's move this method to handleResult (and make it final there). Otherwise we lose the benefit of having a sealed interface pattern matching. Should there be another controller, we would need to include default switch cases to derived methods and lose the security of sealed interfaces